<launch>

  <param name="robot_type" value="j2n6s300" />
  <param name="robot_connected" value="false" />
  <param name="/pick_place_demo/arm/solve_type" value="Manipulation2" />
  <param name="/pick_place_demo/arm/kinematics_solver_timeout" value="0.05" />

  <include file="$(find kinova_gazebo)/launch/robot_launch.launch">
    <arg name="kinova_robotType" value="j2n6s300"/>
  </include>

  <!-- By default, we are not in debug mode -->
  <arg name="debug" default="false" />

  <!-- Run the main MoveIt executable without trajectory execution (we do not have controllers configured by default) -->
  <include file="$(find j2n6s300_moveit_config)/launch/move_group_j2n6s300.launch">
    <arg name="allow_trajectory_execution" value="true"/>
    <arg name="fake_execution" value="false"/>
    <arg name="info" value="true"/>
    <arg name="debug" value="$(arg debug)"/>
    <arg name="joint_states_ns" value="/j2n6s300/joint_states"/>
    <arg name="controller_manager" value="j2n6s300_ros_control"/>
  </include>  

  <!-- ros-control Controller parameters-->  
  <rosparam file="$(find j2n6s300_moveit_config)/config/controllers_ros_control.yaml"/>

  <!-- Run Rviz and load the default config to see the state of the move_group node -->
  <include file="$(find j2n6s300_moveit_config)/launch/moveit_rviz.launch">
    <arg name="config" value="true"/>
    <arg name="debug" value="$(arg debug)"/>
  </include>
</launch>
